manim/mobject/coordinate_systems.py

```diff
@@ -225,6 +225,42 @@ def get_axis_labels(
         )
         return self.axis_labels
 
+    def add_coordinates(
+        self, *axes_numbers: Optional[Iterable[float]], **kwargs
+    ) -> VGroup:
+        """Adds labels to the axes.
+
+        axes_numbers
+            The the numbers to be added to the axes. Use ``None`` to represent an axis without labels.
```

Comment:
Passing None will add the coordinates by default, depending on the range terms of that axis...
```suggestion
            The numbers to be added to the axes. Use ``None`` to represent an axis with default labels.
```